codespell_lib/data/dictionary_rare.txt

```diff
@@ -55,7 +55,10 @@ coo->coup
 copping->coping, copying, cropping,
 corespondent->correspondent
 corespondents->correspondents
-covert->convert
+costumer->customer
+costumers->customers
+covert->convert, cover, covet,
```

Comment:
Not sure about `covert`, even in the rare dictionary, but I guess typos of `convert` might be more common than `covert` itself in the context of code.

Reply:
Is there anything to do here?

Reply:
I was not certain about `covert` appearing at all, even in the **rare** dictionary.